src/js/containers/Buttons.js

```diff
@@ -15,7 +15,7 @@ const mapStateToProps = (state) => {
     var app = {}
     var graphicPresent
     if (activeApp) {
-        subscribedButtons = state.ui[activeApp].subscribedButtons
+        subscribedButtons = state.ui[activeApp].subscribedButtons;
```

Comment:
@LiliiaShlikhtLuxoft Improvement? =)

Reply:
@ValeriiMalkov yes - it's ";" at the end :))